doc/visual-programming/source/widgets/data/datainfo.rst

```diff
@@ -26,23 +26,23 @@ Description
 -----------
 
 A simple widget that presents information on data set size, features,
-targets, meta attributes, and location. It is useful when you manually
-select a subset and you want to see information on the selected subset.
+targets, meta attributes, and location. 
 
-.. figure:: images/data-info4-stamped.png
+.. figure:: images/data-info-stamped.png
 
 1. Information on data set size
-2. Information on discrete and continuous features
+2. Information on discrete and numeric features
```

Comment:
Especially since our File widget says 'Nominal' and 'Numeric', our widget 'Discrete' and 'Continuous', and then there's 'Categorical' and so on. I vote for resolution and standardization.